src/styles/blocks/header.scss

```
@@ -0,0 +1,70 @@
+body {
+  margin: 0;
+  box-sizing: border-box;
+  font-family: "Roboto", serif;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 40px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__img-logo {
```

```suggestion
  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);

&__logo {
display flex
}

  &__img-logo {
```
try this to center the logo picture vertically. Make sure to check if it works